pkg/api/v2/reporter/reporter_suite_test.go

```diff
@@ -1,12 +1,29 @@
 package reporter_test
 
 import (
+	"os"
 	"testing"
 
+	"github.com/solo-io/solo-kit/test/helpers"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+var namespace string
+
+var _ = BeforeSuite(func() {
+	namespace = helpers.RandString(5)
```

Comment:
What's the purpose of the random string? I generally feel that tests should be as deterministic as possible to make errors more reproducible and minimize flakes

Reply:
Yeah good call, I had intially done this since other tests follow a similar pattern. But i don't think it's necessary for these tests so may as well keep it simple